### PR TITLE
[fix/autolayout] Fix auto-layout constraints in ClientItemCell 

### PR DESCRIPTION
**Pull request by felix-schwarz**
## Description
- Fix contraint conflicts in ClientItemCell / ClientQueryViewController
- Replace text-based more dots with actual image to improve performance

## Motivation and Context
Previously, verbose warnings were logged by iOS for every item cell, during regular display as well as drag & drop – likely impacting performance in the process.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: ownCloud/Client/ClientQueryViewController.swift [hunk 1]
```diff
@@ -158,6 +158,8 @@ class ClientQueryViewController: UITableViewController, Themeable, UIDropInterac
 		self.tableView.dragInteractionEnabled = true
 		self.tableView.allowsMultipleSelectionDuringEditing = true
 
+		self.tableView.estimatedRowHeight = 80
```
Comment (hosy) [inline ownCloud/Client/ClientQueryViewController.swift]: can we change 80 to a constant?

File: ownCloud/Client/ClientItemCell.swift [hunk 2]
```diff
@@ -109,14 +103,23 @@ class ClientItemCell: ThemeTableViewCell {
...
 		detailLabel.setContentCompressionResistancePriority(.defaultHigh, for: .vertical)
-		moreButton.setContentHuggingPriority(UILayoutPriority.required, for: .horizontal)
 
 		moreButtonWidthConstraint = moreButton.widthAnchor.constraint(equalToConstant: moreButtonWidth)
 
 		NSLayoutConstraint.activate([
 			iconView.leftAnchor.constraint(equalTo: self.contentView.leftAnchor, constant: horizontalMargin),
 			iconView.rightAnchor.constraint(equalTo: titleLabel.leftAnchor, constant: -spacing),
 			iconView.rightAnchor.constraint(equalTo: detailLabel.leftAnchor, constant: -spacing),
+			iconView.widthAnchor.constraint(equalToConstant: 60),
```
Comment (hosy) [inline ownCloud/Client/ClientItemCell.swift]: can we change 60 to a constant?

File: ownCloud/Client/ClientItemCell.swift [hunk 3]
```diff
@@ -109,14 +103,23 @@ class ClientItemCell: ThemeTableViewCell {
...
+			iconView.centerYAnchor.constraint(equalTo: self.contentView.centerYAnchor),
+
+			titleLabel.rightAnchor.constraint(equalTo: cloudStatusIconView.leftAnchor, constant: -horizontalSmallMargin),
+			detailLabel.rightAnchor.constraint(equalTo: moreButton.leftAnchor, constant: -horizontalMargin),
+
+			titleLabel.topAnchor.constraint(equalTo: self.contentView.topAnchor, constant: horizontalMargin),
+			titleLabel.bottomAnchor.constraint(equalTo: self.contentView.centerYAnchor, constant: -2),
+			detailLabel.topAnchor.constraint(equalTo: self.contentView.centerYAnchor, constant: 2),
```
Comment (hosy) [inline ownCloud/Client/ClientItemCell.swift]: some more numbers we can change to constants

**Comment by felix-schwarz** [comment]
Folded into #347.

**Pull request closed without merging**